templates/Datashades-OpsWorks-CKAN-Instances.cfn.yml.j2

```diff
@@ -303,14 +303,14 @@ Resources:
 
 {% set subnetLetters = ["C","A","B"] %}
 {% for loopingTimes in range(2, 7) %}
-  OpswxEC2Web{{ loopingTimes }}{% if item.tags["PowerManaged"] == "Yes" %}PowerManaged{% else %}AlwaysOn{% endif %}:
+  OpswxEC2Web{{ loopingTimes }}{% if item.tags["PowerManaged"] == "Yes" %}PM{% else %}AlwaysOn{% endif %}:
```

Comment:
This uses abbreviated "PM" but not "AO"?

Reply:
Can add AO

Reply:
What I meant is that it's using "AlwaysOn" instead of "AO".

Reply:
AlwayOn on hostname is too long and can't be read in the console.

Reply:
Have Added